base-cert-manager/certificates.yaml

```diff
@@ -0,0 +1,56 @@
+# https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
```

Comment:
should this name include `cockroachdb` like the other entries in this file?/is anyone likely to have more than one self signed issue per NS?

Reply:
You're right. Thank you